libr/bin/p/bin_xnu_kernelcache.c

```diff
@@ -1882,6 +1902,214 @@ static RStubsInfo *get_stubs_info(struct MACH0_(obj_t) *mach0, ut64 paddr, RKern
 	return stubs_info;
 }
 
+typedef struct {
+	char *name;
+	ut64 size;
+	ut64 meta_va;
+	ut64 supermeta_va;
+} RIOKitClass;
+
+static void resolve_iokit_classes(RVecRBinSymbol *symbols, RBinFile *bf, RKext *kext) {
+	const RVector *sections = MACH0_(load_sections) (kext->mach0);
+	if (!sections) {
+		return;
+	}
+
+	RList *classes = r_list_newf (r_iokit_class_free);
+
+	RBinSymbol *sym;
+	R_VEC_FOREACH (symbols, sym) {
+		if (!strstr (sym->name->oname, ".init."))
+			continue;
+		fprintf (stderr, "\n=== Parsing: %s\n", sym->name->oname);
```

Comment:
```suggestion
		R_LOG_DEBUG ("=== Parsing: %s", sym->name->oname);
```

Reply:
Oops yeah sorry, going to remove this logging when I've finished wiring it up 😊